starport/templates/testutil/stargate/testutil/sample/sample.go.plush

```diff
@@ -0,0 +1,13 @@
+package sample
```

Comment:
Cool idea

Another solution could also have the `AccAddress()` method in a more universal package like `spm`. However, I also like generating this package as a template for the user tests